packages/datagrid/src/textrenderer.ts

```diff
@@ -32,6 +32,11 @@ export class TextRenderer extends CellRenderer {
     this.wrapText = options.wrapText || false;
   }
 
+  /**
+   * Horizontal padding for the text
+   */
+  static H_PADDING = 8;
```

Comment:
Sure, do you want me to also add a member on the TextRendrer class to get the padding from?

Reply:
Yes, I would suggest to follow the pattern of all other properties (such as `font` or `wrapText`); in particular given the explicit naming of `horizontalAlignment` maybe it should be `horizontalPadding`?

Reply:
Or we could just lift this variable to global namespace so that it is not exported and does not introduce a new public API member. I do not have a strong opinion one way or the other.